public/third-party-cookies-check/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Third Party Cookies Test</title>
+  </head>
+
+  <body>
+    <script>
+      const checkCookiesEnable = () => {
+        let isCookieEnabled = window.navigator.cookieEnabled ? true : false
+        if (typeof window.navigator.cookieEnabled == 'undefined' && !isCookieEnabled) {
+          document.cookie = 'testcookie'
+          isCookieEnabled = document.cookie.indexOf('testcookie') != -1 ? true : false
+        }
+
+        return isCookieEnabled
+      }
+
+      ;(function () {
+        window.addEventListener('message', (event) => {
+          try {
+            let data = event.data
+            if (data.test !== 'cookie') return
+            let result = checkCookiesEnable()
+            parent.postMessage(
+              {
+                isCookieEnabled: result,
```

Comment:
Perhaps add a type for this. It seems like you're also checking this in `communicator.ts`?

Reply:
here really I'm not winning anything as this is no TS. In the communicator I only check for the existence of the property. mmm I think is fine like this

Reply:
Ah, I didn't think about that. Let's leave it then.